app/networkapi/settings.py

```diff
@@ -45,7 +46,9 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = env('DEBUG')
 
-ALLOWED_HOSTS = env('ALLOWED_HOSTS')
+ALLOWED_HOSTS = os.getenv("ALLOWED_HOSTS").split(",")
```

Comment:
using `env.ALLOWED_HOSTS` has no fallback safety and will **crash** execution if there is no environment variable by you're telling python to access a nonexistent property. To get around this, you can use a dictionary-safe access like `oi.getenv` instead, which will not crash python, but will give you `None` back if there is no environment variable. Given that we immediately call `split` on the result, and `None.split()` again **does** crash python, providing that fallback value explicitly as a string is super nice.